test/tools/WebListener/Controllers/ResumeController.cs

```diff
@@ -113,7 +113,7 @@ private void SetResumeResponseHeaders()
         private bool TryGetRangeHeader(out string rangeHeader)
         {
             var rangeHeaderSv = new StringValues();
```

Comment:
@xtqqczze For my education - does an analyzer recognize the "var" pattern to convert to 
```c#
            StringValues rangeHeaderSv = new();
```

Reply:
I am not aware of such an analyzer, but one could use [RCS1012](https://github.com/JosefPihrt/Roslynator/blob/master/docs/analyzers/RCS1012.md) codefix first.

Reply:
So we should apply RCS1012 and then apply a fix to the new pattern?